build/cli.version.darwin

```diff
@@ -0,0 +1 @@
+1.0.0.001540
```

Comment:
This is the version it had before, was that wrong? I'm not trying to give it something up to date, just get it ready for the CoreCLR changes.